tests/ltp/ltp-batch2/ltp_disabled_tests.txt

```diff
@@ -800,9 +800,9 @@
 #/ltp/testcases/kernel/syscalls/sendfile/sendfile08
 #/ltp/testcases/kernel/syscalls/sendfile/sendfile09
 #/ltp/testcases/kernel/syscalls/sendmmsg/sendmmsg01
-#/ltp/testcases/kernel/syscalls/sendmsg/sendmsg01
+/ltp/testcases/kernel/syscalls/sendmsg/sendmsg01
```

Comment:
Why are you disabling these test cases? This time these tests failed randomly, next time different tests can fail. I think we shouldn't disable them.

Reply:
I've provided an explanation, and moved the changes to the CI pipeline to a separate PR.